modules/layers/src/base-tile-layer/base-tile-layer.js

```diff
@@ -0,0 +1,156 @@
+import {log} from '@deck.gl/core';
+import {CompositeLayer} from '@deck.gl/core';
+import GeoJsonLayer from '../geojson-layer/geojson-layer';
+import TileCache from './utils/tile-cache';
+
+const defaultProps = {
+  renderSubLayers: {type: 'function', value: props => new GeoJsonLayer(props), compare: false},
+  getTileData: {type: 'function', value: ({x, y, z}) => Promise.resolve(null), compare: false},
+  tile2boundingBox: {
```

Comment:
We usually spell the name out in public APIs (`tileToBoundingBox` or `getTileBoundingBox` to align with the others)

Reply:
I think the reason I did this was because of the former name: https://github.com/uber/deck.gl/pull/4117.  I can change the name of the props in the public API - should I also change the name in `modules/geo-layers/src/tile-layer/utils/tile-util.js` (which is passed in as the prop) or leave it to match the old name, which was `tile2boundingBox` but not exposed as a prop?

Reply:
`tile-util.js` is private so you can rename anything as you please.